tools/KoreBuild.Console/Commands/DockerBuildCommand.cs

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace KoreBuild.Console.Commands
+{
+    internal class DockerBuildCommand : SubCommandBase
+    {
+        private const string DockerIgnore = ".dockerignore";
+        private const string DockerfileExtension = ".dockerfile";
+        private const string Owner = "aspnetbuild";
```

Comment:
Do we already own this docker hub account? Or are you planning on using a private container registry?

Reply:
We should probably claim it if we can. I think it would be useful to take the simple steps to prepare to use a registry of some kind.

Reply:
Squated.